client/webserver/site/src/css/market.scss

```diff
@@ -1,5 +1,4 @@
 div.marketlist {
-  width: 175px;
```

Comment:
I'd much prefer this div to stay a fixed width. The width was chosen because it should accommodate most ticker combos without wrapping. For instance, in the test app server, which has some 4-letter tickers

![image](https://user-images.githubusercontent.com/6109680/112861085-17860100-907a-11eb-81d4-5326d0cb8834.png)

You can see some of the tickers are wrapping, and that the long name DEX is displaying fewer letters.

Wrapping could be solved with `white-space: nowrap`, but that will just result in the ticker text being cut off on the right side, I think. 

This calls for a fixed-width div. If you needed to get it smaller, you could specify a smaller width, logo size, and font size below a certain breakpoint.